javaScriptAJAX-programming/checkResponseStatus/sampleAJAX.js

```diff
@@ -0,0 +1,11 @@
+var xhr = new XMLHttpRequest();
```

Comment:
It's better to use const instead of let to prevent accidental reassignment of variables and promote code stability.

Reply:
Noted on the change request. Code update here 8f9a218463925a3d520e77884e0f4655186a4fe5

For re-review @lycoangelo